recipes/conda-vendor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "conda-vendor" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+  url: https://github.com/MetroStar/conda-vendor/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 2e6db8144ff88f07dcbd0c5f9381d3dd348d3034f3feced895df341373e03081
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
I think 

`script: "{{ PYTHON }} -m pip install . -vv"`

Should be enough, following on the examples https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml

Reply:
thanks! modified per your recommendation